app/script/calling/entities/FlowEntity.js

```diff
@@ -31,6 +31,10 @@ z.calling.entities.FlowEntity = class FlowEntity {
       NEGOTIATION_RESTART_TIMEOUT: 2500,
       SDP_SEND_TIMEOUT: 5 * 1000,
       SDP_SEND_TIMEOUT_RESET: 1000,
+      UPGRADE_TRACK_FAILED: [
```

Comment:
Since those constants are used in a single place in this file, I would rather move their definition in a narrower scope (next to the place they are consumed).